docs/docs/guides/testing.md

```diff
@@ -77,6 +77,87 @@ describe('The form state should', () => {
 });
 ```
 
+## Testing Recoil state with asyncronous default inside of a React component
+
+A common pattern for the default state of atoms is to have an asynchronous function fetch the default state of an atom. However, while testing, the component is immediately suspended upon mounting and will not update in the dom without acting. To test this scenario, we need a helper function:
+
+```jsx
+// testing-utils.js
+import {act} from '@testing-library/react'
```

Comment:
We generally don't include the import statements for the documentation examples, particularly as they can depend on the user's environment.

Reply:
OK. I included them in my example since the rest of the file included them. I can make things consistent throughout the file!